nbs/tutorials/tutorial.ipynb

```diff
@@ -64,30 +64,32 @@
     "4. nbdev\n",
```

Comment:
<p>Pass the <code>-y</code> flag here too? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Done</p>